manifests/controller-manager/vsphere-cloud-controller-manager-ds.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       serviceAccountName: cloud-controller-manager
       containers:
         - name: vsphere-cloud-controller-manager
-          image: gcr.io/cloud-provider-vsphere/vsphere-cloud-controller-manager:latest
+          image: gcr.io/cloud-provider-vsphere/cpi/release/manager:latest
```

Comment:
@frapposelli So here is where I need your input for sure. What I did here was point to the latest *release* -- which won't exist once this merges until we tag a new release. To preserve the existing behavior, we would instead `s/release/ci`, which would always point to the latest build from the tip of `master`, which is actually what we are doing now.

So that begs a question -- what do we want our manifests to point users to by default? The latest *release*, or the latest build from master? I would argue for the latest release, but if we do that, then we have this temporary situation where once this goes in, there is no release to point to for some time.

That's why I was thinking about changing the manifests later -- it's okay to point to an older version until we are happy with this new tooling/release structure. Thoughts?

Reply:
`latest` should be the latest tagged version, and should be the one referenced in the manifest.
`ci` or `<sha id>` should be the latest build from master, available for people to use but not reference in the examples.

Since we're almost ready to tag a new release, I'd like to keep refining this PR and then tag a new version as soon as we merge, this would put the new manifests in effect right away.

Reply:
Agreed. In fact, what you will be able to do is use either `release/manager:latest`, which will be the latest "official release", OR use `ci/manager:latest` which is the most recent build from master. I like that they are separate. :) So, from your comments, I think I what I have in the manifests now is correct.

Reply:
LGTM 👍